Edit the results section of step-61.

bangerth opened the pull request:
Follow-up to #7953 and, of course, @sophy1029's #6455.

drwells commented on examples/step-61/doc/results.dox:
```diff
@@ -80,13 +88,20 @@ to divide each cell interior into 4 subcells.
+combination, the solution is now substantially more accurate and, in
+particular so close to being continuous at the interfaces that we can
+no longer distinguish the interface pressures $p^\partial$ from the
+interior pressures $p^\cir$ on the adjacent cells.
```
```suggestion
interior pressures $p^\circ$ on the adjacent cells.
```

drwells commented on examples/step-61/doc/results.dox:
```diff
@@ -101,31 +116,41 @@ These are the convergence rates of pressure, velocity and flux on $\mbox{WG}(Q_1
+The following are interior pressures and face pressures implemented using 
+$WG(Q_2,Q_2;RT_{[2]})$, with mesh size $h = 1/32$ (i.e., 5 global mesh
+refinement steps). In the program, we use
+`data_out_face.build_patches (fe.degree)` when generating graphical output
```
```suggestion
`data_out_face.build_patches(fe.degree)` when generating graphical output
```

drwells commented on examples/step-61/doc/results.dox:
```diff
@@ -1,20 +1,25 @@
-velocity and flux should be around 1 on $\mbox{WG}(Q_0,Q_0;RT_{[0]})$ , 2 on 
-$\mbox{WG}(Q_1,Q_1;RT_{[1]})$, and 3 on $\mbox{WG}(Q_2,Q_2;RT_{[2]})$.
+We run the program with a right hand side that will produce the solution
+$p = \sin(\pi x) \sin(\pi y)$ and with homogenous Dirichlet 
```
```suggestion
$p = \sin(\pi x) \sin(\pi y)$ and with homogeneous Dirichlet 
```
(I only noticed this because of #7979)

bangerth replied:
Ha -- I think that both spellings are correct. But I'm happy to go with the generally used one.

drwells commented:
We will also need to rebase (but we can ignore the changes on `master`).

I am not sure why but on my machine the equations aligned in this PR do not show up as aligned:
![Screenshot_2019-04-29 The deal II Library The step-61 tutorial program](https://user-images.githubusercontent.com/3917035/56938305-2602c100-6acf-11e9-8903-b370cce78525.png)
Does the alignment work for you?

bangerth commented:
I've taken your suggestions and rebased. I don't know about the alignment, but can imagine that it's because of the use of the old-fashioned `eqnarray`. I've changed it to the more moden `align`. The latter requires the use of only one `&` alignment character, whereas the former requires two -- which may have been the cause.

Let's not agonize over this. If it turns out to still look odd in the online version, we can always follow up.

drwells commented:
I agree (and the equations are now aligned).

drwells commented:
Since this only changes `.dox` files lets merge.
